app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function should include input validation to ensure the command string is split into exactly three parts: the command 'cp', the source file name, and the destination file name. Consider adding a check to verify the length of `command_list`.